src/App/Pages/Send/SendAddEditPage.xaml.cs

```diff
@@ -111,11 +111,17 @@ protected override void OnDisappearing()
         private void TextType_Clicked(object sender, EventArgs eventArgs)
         {
             _vm.TypeChanged(SendType.Text);
+            _nameEntry.ReturnType = ReturnType.Next;
+            _nameEntry.ReturnCommand = new Command(() => _textEditor.Focus());
+            RequestFocus(_nameEntry);
```

Comment:
Would we want this focused "above" the fold when the type is changed, but now since Name is above the Type, I would think changing the type shouldn't also change or re-focus the Name field, it should "in theory" already be populated and unchanging.

Reply:
The physical touch of the type buttons grabs focus from the Name field, and this puts it back.  :)

Reply:
fixed per discussion, now checking for empty field before refocus